korman/properties/modifiers/gui.py

```diff
@@ -330,3 +333,316 @@ def _create_moul_nodes(self, clickable_object, nodes, journalnode, age_name):
+                                    ("FishBookLinkOut", "Kneeling", "The avatar kneels in front of the book and places their hand on the panel"),],
+                             default="LinkOut",
+                             options=set())
+
+    # Interactables
+    seek_point = PointerProperty(name="Seek Point",
+                                 description="The point the avatar will seek to before opening the Linking Book GUI",
+                                 type=bpy.types.Object,
+                                 poll=idprops.poll_empty_objects)
+    clickable_region = PointerProperty(name="Clickable Region",
+                                       description="The region in which the avatar must be standing before they can click on the Linking Book",
+                                       type=bpy.types.Object,
+                                       poll=idprops.poll_mesh_objects)
+    clickable = PointerProperty(name="Clickable",
+                                description="The object the avatar will click on to activate the Linking Book GUI",
+                                type=bpy.types.Object,
+                                poll=idprops.poll_mesh_objects)
+
+    # -- Path of the Shell options --
+    # Popup Appearance
+    book_cover_image = PointerProperty(name="Book Cover",
+                                       description="Image to use for the Linking Book's cover (Optional: book starts open if left blank)",
+                                       type=bpy.types.Image,
+                                       options=set())
+    link_panel_image = PointerProperty(name="Linking Panel",
+                                       description="Image to use for the Linking Panel",
+                                       type=bpy.types.Image,
+                                       options=set())
+    stamp_image = PointerProperty(name="Stamp Image",
+                                  description="Image to use for the stamp on the page opposite the book's linking panel, if any",
+                                  type=bpy.types.Image,
+                                  options=set())
+    stamp_x = IntProperty(name="Stamp Position X",
+                          description="X position of Stamp",
+                          default=140,
+                          subtype="UNSIGNED")
+    stamp_y = IntProperty(name="Stamp Position Y",
+                          description="Y position of Stamp",
+                          default=255,
+                          subtype="UNSIGNED")
+
+    def export(self, exporter, bo, so):
+        our_versions = (globals()[j] for j in self.versions)
+        version = exporter.mgr.getVer()
+        if version not in our_versions:
+            # We aren't needed here
+            exporter.report.port("Object '{}' has a LinkingBookMod not enabled for export to the selected engine.  Skipping.",
+                                 bo.name, version, indent=2)
```

Comment:
This will raise an exception due to too many format arguments. I would probably leave out `version` because it's an integer.

Reply:
Do we have an accessible method to get the game version name from a version integer value?

Reply:
I don't think so